resources/user-agents/crawlers/google/google-adsense.json

```diff
@@ -54,6 +54,24 @@
           "properties": {
             "Version": "2.1"
           }
+        },
+        {
+          "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*Mediapartners-Google) Chrome/* Safari/*",
+          "devices": {
```

Comment:
Should we detect the device at all in a bot?

Certainly the bot isn't running on an actual Samsung phone is it?

Reply:
I asume they send the device information to see if the website delivers different content for different devices. This is not the only bot where we are detecting a device.

Reply:
Searched our logs for the past 30 days, these are the only two UAs with "Mediapartners-Google" in them.

`Mozilla/5.0 (Linux; Android 4.0.4; Galaxy Nexus Build/IMM76B) AppleWebKit/537.36 (KHTML, like Gecko; Mediapartners-Google) Chrome/41.0.2272.118 Mobile Safari/537.36`

and

`Mozilla/5.0 (Macintosh; Intel Mac OS X 10_7_3) AppleWebKit/537.36 (KHTML, like Gecko, Mediapartners-Google) Chrome/41.0.2272.118 Safari/537.36`

Just FYI.  

I don't think the device information is important for bots, but no harm I suppose.